.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,28 @@
-## Summary
 
-*Short description*
+# Checklist
 
-## TODO
+- [ ] Target milestone
+- [ ] New tests
```

Comment:
Suggesting to call it "Test coverage satisfactory" - such that OP can indicate that "Yes I'm aware that test coverage is down, but I have reasons". Once we approach high coverage rates like 80-90%, it will be normal that PRs make things fluctuate around the peak.